verifyml/model_tests/FEAT/MinMaxMetricThreshold.py

```diff
@@ -28,35 +29,37 @@
 @dataclass
 class MinMaxMetricThreshold(ModelTest):
     """Test if all the subgroups for a given attribute meets a certain level of expected,
-    performance. If fpr / fnr is used as a metric, they have to be lower than the threshold.
-    If tpr / tnr is used as a metric, they have to be greater than the threshold.
+    performance.
 
     The test also stores a dataframe showing the results of each groups and ROC curve plots
-    for every subgroup along with the points which maximises tpr-fpr.
+    (for classification problem) for every subgroup along with the points which maximises tpr-fpr.
 
     Args:
       attr: Column name of the protected attribute.
-      metric: Type of performance metric for the test, choose from 'fpr' - false positive rate,
-        'tpr' - true positive rate, 'fnr' - false negative rate, 'tnr' - true negative rate.
-      threshold: Threshold for the test. To pass, fpr / fnr has to be lower than the threshold or tpr/tnr
-        has to be greater than the threshold.
-      proba_threshold: Probability threshold for the output to be classified as 1. By default, it is 0.5.
-      test_name: Name of the test, default is 'ROC/Min Max Threshold Test'.
+      metric: Type of performance metric for the test, 
+         For classification problem, choose from 'fpr' - false positive rate,
+         'tpr' - true positive rate, 'fnr' - false negative rate, 'tnr' - true negative rate.
+         For regression problem, choose from 'mse' - mean squared error.
+      threshold: Threshold for the test. To pass, fpr/fnr/mse has to be lower than the threshold or tpr/tnr
+         has to be greater than the threshold.
+      proba_threshold: Arg for classification problem, probability threshold for the output to be classified as 1. 
+         By default, it is 0.5.
+      test_name: Name of the test, default is 'Min Max Threshold Test'.
       test_desc: Description of the test. If none is provided, an automatic description
          will be generated based on the rest of the arguments passed in.
     """
 
     attr: str
-    metric: Literal["fpr", "tpr", "fnr", "tnr"]
+    metric: Literal["fpr", "tpr", "fnr", "tnr", "mse"]
     threshold: float
     proba_threshold: float = 0.5
     plots: dict[str, str] = field(repr=False, default_factory=dict)
-    test_name: str = "ROC/Min Max Threshold Test"
+    test_name: str = "Min Max Threshold Test"
     test_desc: str = None
 
     def __post_init__(self):
-        metrics = {"fpr", "tpr", "fnr", "tnr"}
-        if self.metric in ["fpr", "fnr"]:
+        metrics = {"fpr", "tpr", "fnr", "tnr", "mse"}
```

Comment:
Suggested replacement for lines 61-67 so that you only write the metrics once, fewer chances for errors / typos:
```
lower_req_metrics = {"fpr", "fnr", "mse"}
higher_req_metrics = {"tpr", "tnr"}
metrics = lower_req_metrics | higher_req_metrics 

if self.metric not in lower_req_metrics | higher_req_metrics:
  raise ValueError blabla

req = "lower" if self.metric in lower_req_metrics else "higher"
```